src/net.cpp

```diff
@@ -2813,35 +2811,36 @@ bool CConnman::NodeFullyConnected(const CNode* pnode)
 
 void CConnman::PushMessage(CNode* pnode, CSerializedNetMsg&& msg)
 {
-    size_t nMessageSize = msg.data.size();
-    LogPrint(BCLog::NET, "sending %s (%d bytes) peer=%d\n",  SanitizeString(msg.m_type), nMessageSize, pnode->GetId());
+    const size_t message_size{msg.data.size()};
+    LogPrint(BCLog::NET, "sending %s (%d bytes) peer=%d\n", SanitizeString(msg.m_type), message_size, pnode->GetId());
 
     // make sure we use the appropriate network transport format
-    std::vector<unsigned char> serializedHeader;
-    pnode->m_serializer->prepareForTransport(msg, serializedHeader);
-    size_t nTotalSize = nMessageSize + serializedHeader.size();
+    std::vector<unsigned char> serialized_header;
+    pnode->m_serializer->prepareForTransport(msg, serialized_header);
+    const size_t total_size{message_size + serialized_header.size()};
 
-    size_t nBytesSent = 0;
+    bool optimistic_send{false};
     {
         LOCK(pnode->cs_vSend);
-        bool optimisticSend(pnode->vSendMsg.empty());
+        optimistic_send = pnode->vSendMsg.empty();
 
         //log total amount of bytes per message type
```

Comment:
If you retouch the branch, you may as well touch _every_ line in this function:

```suggestion
        // Log total amount of bytes per message type
```